bower.json

```diff
@@ -10,7 +10,7 @@
   ],
   "description": "Library to pass Sass variables via CSS to JavaScript",
   "main": [
-    "js/dist/sass-to-js.min.js",
+    "js/src/sass-to-js.js",
```

Comment:
Why this change is needed?
Usually the dist/ version is used, e.g. for jQuery: https://github.com/jquery/jquery/blob/master/package.json